### PR TITLE
Fdhd 101 selective reingestion

### PR DESCRIPTION
**Pull request by pmarchini**
## Pull Request Type

- [X] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [ ] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [ ] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [ ] No sensible content has been committed

### CODE REVIEW

File: docs/fast_data/bucket_storage_support/configuration/ingestion_reloader.md [hunk 1]
```diff
@@ -198,20 +205,96 @@ paths:
...
-
-This route allows the user to stop the ongoing reingestion of a topic.
-The user has to specify the _ingestion_ topic that is currently being reingested.
+This route allows the user to stop the ongoing reingestion of a topic. The user has to specify the ingestion topic that is currently being reingested.
 
 :::caution
 This route can stop only the reingestion of a topic, not the one of a single file.
 :::
+
+Please refer to the [Custom Selector Feature section](#customer-selectors) for more details on how to use the custom selector feature for precise and targeted filtering of messages to be reingested.
```
Comment (TheRusCompass) [inline docs/fast_data/bucket_storage_support/configuration/ingestion_reloader.md:220]:
```suggestion
Please refer to the [Custom Selector Feature section](#customer-selectors) for more details on how to use the custom selector feature for precise and targeted filtering of messages to be reingested.

```

File: docs/fast_data/bucket_storage_support/configuration/ingestion_reloader.md [hunk 2]
```diff
@@ -198,20 +205,96 @@ paths:
...
+## Customer Selectors
+
+The customSelector feature is a powerful addition to the Ingestion Reloader service that allows users to apply custom filtering logic to select specific messages from the bucket for re-ingestion into the Kafka topic.
```
Comment (TheRusCompass) [inline docs/fast_data/bucket_storage_support/configuration/ingestion_reloader.md:223]: I would provide a quick example here or a brief explanation of a possible use case, to let the user better  understand the feature

Comment (danibix95) [reply docs/fast_data/bucket_storage_support/configuration/ingestion_reloader.md:223]: Maybe it is possible to provide a cURL as an example?

**Review by danibix95 — COMMENTED** [review]
Great job!

File: docs/fast_data/bucket_storage_support/configuration/ingestion_reloader.md [hunk 3]
```diff
@@ -220,7 +220,40 @@ This route can stop only the reingestion of a topic, not the one of a single fil
...
+
+MarketingSelector()
+You can trigger the re-ingestion with a cURL command:
+```
+
+
+Here's an example of a `cURL` command to trigger the re-ingestion process using the custom selector script:
+
+```bash
+curl -X POST "http://localhost:3000/reingestion/topic" \
```
Comment (TheRusCompass) [inline docs/fast_data/bucket_storage_support/configuration/ingestion_reloader.md:248]: I would not use localhost in the example but I would put something like <your_address> or something similar

**Review by TheRusCompass — APPROVED** [review]

**Pull request merged**